### PR TITLE
fix: identifer

### PR DESCRIPTION
**Pull request by sean-brydon**
Fixes verify

### CODE REVIEW

File: packages/trpc/server/routers/viewer/auth/_router.tsx [hunk 1]
```diff
@@ -71,22 +71,25 @@ export const authRouter = router({
...
+      }
+
+      // Unreachable code but required for type safety
+      if (!UNSTABLE_HANDLER_CACHE.sendVerifyEmailCode) {
+        throw new Error("Failed to load handler");
+      }
+
+      return UNSTABLE_HANDLER_CACHE.sendVerifyEmailCode({
+        input,
+        req,
```
Comment (sean-brydon) [inline packages/trpc/server/routers/viewer/auth/_router.tsx:90]: Add req to pass through

File: packages/trpc/server/routers/viewer/auth/sendVerifyEmailCode.handler.ts [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
 import type { TSendVerifyEmailCodeSchema } from "./sendVerifyEmailCode.schema";
 
 type SendVerifyEmailCode = {
   input: TSendVerifyEmailCodeSchema;
+  req: NextApiRequest | undefined;
 };
 
-export const sendVerifyEmailCodeHandler = async ({ input }: SendVerifyEmailCode) => {
+export const sendVerifyEmailCodeHandler = async ({ input, req }: SendVerifyEmailCode) => {
+  const identifer = req ? getIP(req) : input.email;
```
Comment (sean-brydon) [inline packages/trpc/server/routers/viewer/auth/sendVerifyEmailCode.handler.ts:15]: Req shouldnt be undefined but can be on a type level

File: packages/trpc/server/routers/viewer/auth/sendVerifyEmailCode.handler.ts [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+
+  await checkRateLimitAndThrowError({
+    rateLimitingType: "core",
+    identifier: `emaiLVerifyByCode.${identifer}`,
```
Comment (keithwillcode) [inline packages/trpc/server/routers/viewer/auth/sendVerifyEmailCode.handler.ts:19]: L is capitalized

**Review by keithwillcode — DISMISSED** [review]
Nice thanks for submitting so quickly. Just a small typo

**Review by Udit-takkar — APPROVED** [review]
LGTM. (Fixed type error)

**Pull request merged**